### ISSUE TITLE
"sharing" doesn't work as expected 🤔

### ISSUE THREAD

**Issue by Rowin63** [osmandapp/OsmAnd#21628]
### Description

When sharing ONE track (3dots > share) it's possible to choose the app of your choice (mail, WhatsApp, others...)
But if MORE than one track is selected, it's only possible to use Osmands "export" option. 

### Steps to reproduce

Try to SHARE more than one selected track 

### Actual result

"Export" menu opens 

### Expected result

Usual "share" options available 

### Your Environment (required)

Osmand +4.9.10

**Comment by yuriiurshuliak** [comment]
This behavior can be reproduced by following these steps:
1. Open OsmAnd and go to **Menu > My Places > Tracks**.
2. Tap the **three dots** in the top-right corner and select **Select**.
3. Choose **more than two tracks**.
4. Tap the **three dots** again and select **Share**.

**Actual Result:**  
The export context menu opens.

**Expected Result:**  
The usual "Share" menu should open, allowing selection of apps to share the tracks (e.g., mail, WhatsApp).

OsmAnd~ 5.0.0#4010m, Released: 2024-12-27

https://github.com/user-attachments/assets/4f2ac8bf-b63c-4cc7-8bea-5fd184a8cdbd

**Comment by vshcherb** [comment]
That's correct, sharing multiple files had some limitations, so it's combined into 1 file.
